cmd/broadcast/main.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"flag"
+	"github.com/harmony-one/libunison/internal/ida/coopcast"
+	"github.com/harmony-one/libunison/internal/ida/manycast"
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"net"
+	"strconv"
+	"time"
+)
+
+// InitCoopCastNode creates a coopcast node
+func InitCoopCastNode(confignbr string, configallpeer string, t0 float64, t1 float64, t2 float64, base float64, hop int) *coopcast.Node {
+	rand.Seed(time.Now().UTC().UnixNano())
+	config1 := NewConfig()
+	err := config1.ReadConfigFile(confignbr)
+	if err != nil {
+		log.Printf("unable to read config file %v", confignbr)
+	}
+	selfPeer, peerList, _ := config1.GetPeerInfo()
+	config2 := NewConfig()
+	err = config2.ReadConfigFile(configallpeer)
+	if err != nil {
+		log.Printf("unable to read config file %v", configallpeer)
+	}
+
+	_, _, allPeers := config2.GetPeerInfo()
+	Cache := make(map[coopcast.HashKey]*coopcast.RaptorQImpl)
```

Comment:
Although I do not see a definitive rule in Effective Go, I believe the convention is to lowercase local variable names.